storage/storage.go

```diff
@@ -119,7 +119,6 @@ type Client struct {
 	tc storageClient
 	// useGRPC flags whether the client uses gRPC. This is needed while the
 	// integration piece is only partially complete.
-	// TODO: remove before merging to main.
```

Comment:
Could you also remove the field (`useGRPC`) which it is referencing?

Looks like the only reference is on L240 of this file.